lib/Makefile.am

```diff
@@ -150,6 +150,7 @@ if LIBMESH_ENABLED
   ../src/IB/CIBFEMethod.cpp \
   ../src/IB/IBFECentroidPostProcessor.cpp \
   ../src/IB/IBFEMethod.cpp \
+  ../src/IB/IBFEInstrumentPanel.cpp \
```

Comment:
IBFEInstrumentPanel.cpp should go above IBFEMethod.cpp (try to keep everything in alphabetical order).